source/fundamentals/crud/write-operations/update.txt

```diff
@@ -0,0 +1,293 @@
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = await _restaurantsCollection.UpdateManyAsync(filter, update);
+
+   .. tab:: Synchronous
+      :tabid: update-many-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = _restaurantsCollection.UpdateMany(filter, update);
+
+.. tip::
+
+   Find runnable examples that use these methods under :ref:`Additional
+   Information <csharp-update-info>`.
+
+Customize the Update Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you need to customize your update operation, create an ``UpdateOptions`` object 
+and pass it to ``UpdateOne()`` or ``UpdateMany()``. The ``UpdateOptions`` type lets 
+you configure the following properties:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``ArrayFilters``
+     - | Specifies which array elements to modify for an update operation on an array field.
+         See :manual:`the MongoDB server manual</reference/command/update/#update-elements-match-arrayfilters-criteria>`
+         for more information.
+
+   * - ``BypassDocumentValidation``
+     - | Specifies whether the update operation bypasses document validation. This lets you 
+         update documents that don't meet the schema validation requirements, if any 
+         exist. See :manual:`the MongoDB server manual</core/schema-validation/#schema-validation>`
+         for more information on schema validation.
+
+   * - ``Collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. See :manual:`the MongoDB server manual</reference/collation/#std-label-collation>`
+         for more information on collation.
+
+   * - ``Comment``
+     - | Gets or sets the user-provided comment for the operation. 
+         See :manual:`the MongoDB server manual</reference/command/update/#command-fields>`
+         for more information.
+
+   * - ``Hint``
+     - | Gets or sets the index to use to scan for documents. 
+         See :manual:`the MongoDB server manual<reference/command/update/#std-label-update-command-hint>`
+         for more information.
+
+   * - ``IsUpsert``
+     - | Specifies whether the update operation performs an upsert operation if no 
+         documents match the query filter. 
+         See :manual:`the MongoDB server manual </reference/command/update/#std-label-update-command-upsert>`
+         for more information.
+
+   * - ``Let``
+     - | Gets or sets the let document. 
+         See :manual:`the MongoDB server manual </reference/command/update/#std-label-update-command-upsert>`
+         for more information.
+
+.. tip::
+
+    The ``UpdateOptions`` parameter is optional. Use it only if you must change the
+    default update behavior.
+
+Return Value
+~~~~~~~~~~~~
+
+The ``UpdateOne()`` and ``UpdateMany()`` methods each return an ``UpdateResult`` 
+object. The ``UpdateResult`` type contains the following properties:
```

Comment:
```suggestion
type. The ``UpdateResult`` type contains the following properties:
```

Reply:
I thought about this one while writing. Is it correct to say it "returns a type"? Technically speaking, it returns an object _of_ a particular type. I'd prefer to keep this the way it is unless you feel strongly.

Reply:
yeah, it's a funny distinction. the UpdateResult class/type contains all the properties in that list (i.e., they are defined within the class), but the object you get back from the operation may not (since one or more may not be relevant in a given case).